engine/runtime/src/main/java/org/enso/interpreter/runtime/Module.java

```diff
@@ -557,7 +556,7 @@ public final ModuleScope getScope() {
    */
   final ModuleScopeBuilder getScopeBuilder(boolean reset) {
     var sb = TruffleCompilerContext.findCompilerModule(this).getScopeBuilder(reset);
-    return sb.unsafeScopeBuilder();
```

Comment:
- I have to admit I don't understand the Scala(?) traditional of calling some methods _"unsafe"_. 
- Probably a substitute for missing `throws` declarations?
- In Java one would `throws ClassCastException` and document why...